.cirrus.yml

```diff
@@ -0,0 +1,38 @@
+# By default, Cirrus mounts an empty volume to `/tmp`
+# which triggers all sorts of warnings like "system temporary path is world-writable: /tmp".
+# Lets workaround it by specifying a custom volume mount point.
+env:
+  CIRRUS_VOLUME: /cirrus
+
+task:
+  name: Test on Graviton2 ($CC)
```

Comment:
Could you change the name like below?

```
name: Test on Arm64 Graviton2 ($CC)
```

I referred the following documents. The "Arm64 Graviton2" is used as a name in Travis. I would like to align with it.

* https://docs.travis-ci.com/user/multi-cpu-architectures
  > Arm-based building on Arm64 Graviton2 CPU now supports both Open Source and commercial projects. The total concurrency capacity is limited, but may adjusted based on the demand.
* https://aws.amazon.com/ec2/graviton/

Reply:
Changed in 322c71550c5a9baf5bdee52205080cd4d1effac7